### PR TITLE
[DONE] Improve Concurrent Mode API Ref

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

File: content/docs/concurrent-mode-reference.md [hunk 1]
```diff
@@ -71,11 +72,11 @@ ReactDOM.createBlockingRoot(rootNode).render(<App />)
 
 `Suspense` يتيح لمكوناتك "الانتظار" لشيء ما قبل أن تتمكن من التصيير، مع إظهار احتياطي أثناء الانتظار.
```
Comment (3imed-jaberi) [inline content/docs/concurrent-mode-reference.md:73]: `fallback === احتياطي ` > confused 😕 !!
Here, I think the core doc lacks the word `component` or `component` beside `fallback`.

@Aissaoui-Ahmed  @Fcmam5 @asantarissy 

```suggestion
`Suspense` يتيح لمكوناتك "الانتظار" لشيء ما قبل أن تتمكن من التصيير، مع إظهار احتياطي (fallback) أثناء الانتظار.
```

Comment (Fcmam5) [reply content/docs/concurrent-mode-reference.md:73]: I using the original word between parentheses, so I'd go & commit your suggestion

File: content/docs/concurrent-mode-reference.md [hunk 2]
```diff
@@ -15,17 +15,18 @@ prev: concurrent-mode-adoption.html
...
+>تصف هذه الصفحة **الميزات التجريبية التي [ليست متوفرة بعد](/docs/concurrent-mode-approach.html) في الإصدار المستقر**. لا تعتمد على البناءات React التجريبية في تطبيقات الإنتاج. قد تتغير هذه الميزات بشكل كبير ودون سابق إنذار قبل أن تصبح جزءًا من React.
 >
 >تستهدف هذه المستندات إلى المُتَبنّين المبكرين والأشخاص الفضوليين. **إذا كنت جديدًا في React، فلا تقلق بشأن هذه الميزات** --  لست بحاجة إلى تعلمها الآن.
 
 </div>
 
 هذه الصفحة هي مرجع واجهة برمجة التطبيقات لـ React [الوضع المتزامن](/docs/concurrent-mode-intro.html). إذا كنت تبحث عن مقدمة موجهة بدلاً من ذلك، فتحقق من [أنماط واجهة المستخدم المتزامنة](/docs/concurrent-mode-patterns.html).
 
-**ملاحظة: هذه هي معاينة المجتمع وليست الإصدار النهائي المستقر. من المحتمل أن تكون هناك تغييرات مستقبلية على واجهات برمجة التطبيقات تلك. استخدام على مسؤوليتك الخاصة!**
+**ملاحظة: هذه عينة توفر نظرة عامة للمجتمع وليست الإصدار النهائي المستقر. من المحتمل أن تكون هناك تغييرات مستقبلية على واجهات برمجة التطبيقات تلك. استخدم على مسؤوليتك الخاصة!**
```
Comment (Fcmam5) [inline content/docs/concurrent-mode-reference.md:27]: I love this choice of words, thank you Imed!

File: content/docs/concurrent-mode-reference.md [hunk 3]
```diff
@@ -34,15 +35,15 @@ prev: concurrent-mode-adoption.html
...
+## تفعيل الوضع المتزامن {#concurrent-mode}
 
 ### `createRoot` {#createroot}
 
 ```js
 ReactDOM.createRoot(rootNode).render(<App />);
 ```
 
-تستبدل `ReactDOM.render(<App />, rootNode)` وتُمَكّن الوضع المتزامن.
+إستبدل `ReactDOM.render(<App />, rootNode)` وتُفعل الوضع المتزامن.
```
Comment (Fcmam5) [inline content/docs/concurrent-mode-reference.md:46]: I recommend using tashkeel, especially when using المبني للمجهول 

```suggestion
إستبدل `ReactDOM.render(<App />, rootNode)` وتُفعِّل الوضع المتزامن.
```

**Pull request merged**